app/settings/log.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+#
+# Copyright 2014 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FI-WARE project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+#
+import settings
+import logging
+
+__author__ = 'fla'
+
+logger = logging.getLogger('GlanceSync')
+logger.setLevel(settings.LOGGING_LEVEL)
+fh = logging.FileHandler(settings.LOGGING_PATH + '/GlanceSync.log')
+fh.setLevel(settings.LOGGING_LEVEL)
+formatter = logging.Formatter('%(asctime)s %(levelname)s glancesync [-] %(message)s')
+fh.setFormatter(formatter)
+logger.addHandler(fh)
```

Comment:
Given we already have a `*.cfg` configuration file with different sections, we could put there logging-specific sections describing appenders / handlers / formatters... and then invoke `logging.config.fileConfig()` (see [logging](https://docs.python.org/2/library/logging.config.html#configuration-functions))